src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Providers/TemplateCollectionProviderTests.cs

```diff
@@ -85,10 +87,11 @@ public static IEnumerable<object[]> GetImageInfoForArtifactWithTag()
 
         public static IEnumerable<object[]> GetDefaultTemplatesInfo()
         {
-            yield return new object[] { "microsofthealth/fhirconverter:default", 852 };
-            yield return new object[] { "microsofthealth/hl7v2templates:default", 852 };
+            yield return new object[] { "microsofthealth/fhirconverter:default", 915 };
+            yield return new object[] { "microsofthealth/hl7v2templates:default", 915 };
```

Comment:
Do we need to change these two lines?

Reply:
After offline discussion, we can update this value regularly when templates are stable.